src/plugins/management/public/management_sections_service.test.ts

```diff
@@ -105,4 +106,29 @@ describe('ManagementService', () => {
       ]
     `);
   });
+
+  it('should disable apps register in opensearchDashboards section', () => {
```

Comment:
Could you help explain how this is testing the "opensearch-dashboards" translation code?

Reply:
In the capability provider, the capability key is `opensearchDashboards`, when registering the default section `opensearchDashboards: this.registerSection(OpenSearchDashboardsSection)`, the section id is defined as `opensearch-dashboards` and this id was used to retrieve the app status register with capability key `opensearchDashboards`.

So we need to do the mapping from `opensearch-dashboards` -> `opensearchDashboards`